pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/guide.en-us.md

```diff
@@ -0,0 +1,1248 @@
+```
+If the output looks like above, then you configured the S3 target object successfully.
+
+**Hint:**
+In case the target object fails to become healthy, you can inspect the logs from the `trilio-ovh-s3-target-validator` Pod to find the issue:
+
+First, you need to find the target validator
+
+```shell
+kubectl get pods -n tvk | grep trilio-ovh-s3-target-validator
+```
+The output looks similar to:
+
+```text
+trilio-ovh-s3-target-validator-tio99a-6lz4q	1/1     Running     0          104s
+```
+Now, fetch logs data
+
+```shell
+kubectl logs pod/trilio-ovh-s3-target-validator-tio99a-6lz4q -n tvk
+```
+The output looks similar to (notice the exception as an example):
+
+```text
+...
+INFO:root:2021-11-24 09:06:50.595166: waiting for mount operation to complete.
+INFO:root:2021-11-24 09:06:52.595772: waiting for mount operation to complete.
+ERROR:root:2021-11-24 09:06:54.598541: timeout exceeded, not able to mount within time.
+ERROR:root:/triliodata is not a mountpoint. We can't proceed further.
+Traceback (most recent call last):
+  File "/opt/tvk/datastore-attacher/mount_utility/mount_by_target_crd/mount_datastores.py", line 56, in main
+    utilities.mount_datastore(metadata, datastore.get(constants.DATASTORE_TYPE), base_path)
+  File "/opt/tvk/datastore-attacher/mount_utility/utilities.py", line 377, in mount_datastore
+    mount_s3_datastore(metadata_list, base_path)
+  File "/opt/tvk/datastore-attacher/mount_utility/utilities.py", line 306, in mount_s3_datastore
+    wait_until_mount(base_path)
+  File "/opt/tvk/datastore-attacher/mount_utility/utilities.py", line 328, in wait_until_mount
+    base_path))
+Exception: /triliodata is not a mountpoint. We can't proceed further.
+...
+```
+Next, you will discover the TVK web console which is a really nice and useful addition to help you manage backup and restore operations very easy, among many others.
+
+## Step 3 - Getting to Know the TVK Web Management Console
+
+While you can manage backup and restore operations from the `CLI` entirely via `kubectl` and `CRDs`, `TVK` provides a [Web Management Console](https://docs.trilio.io/kubernetes/management-console/user-interface) to accomplish the same operations via the GUI. The management console simplifies common tasks via point and click operations, provides better visualization and inspection of TVK cluster objects, as well as to create disaster recovery plans (or `DRPs`).
+
+The Helm based installation covered in [Step 1 - Installing TrilioVault for Kubernetes](#step-1---installing-triliovault-for-kubernetes) already took care of installing the required components for the web management console.
+
+### Getting Access to the TVK Web Management Console
+
+To be able to access the console and explore the features it offers, you can either user LoadBalancer, NodePort or need to port forward the ingress gateway service for TVK.
+
+First, you need to identify the `ingress-gateway` service from the `tvk` namespace:
```

Comment:
replace 
ingress-gateway
with
ingress-nginx-controller

Reply:
Fixed the changes